src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
Doesn't it disable logging in the production build? Maybe the same thing should happen here. A log isn't bad for devs, but shouldn't make its way to production.